presto-main/src/main/java/io/prestosql/operator/ExchangeClientConfig.java

```diff
@@ -83,13 +85,38 @@ public Duration getMaxErrorDuration()
         return maxErrorDuration;
     }
 
+    @Config("exchange.timeout-failure-detection")
+    public ExchangeClientConfig setTimeoutFailureDetection(boolean b)
+    {
+        this.timeoutFailureDetection = b;
+        return this;
+    }
+
+    public boolean getTimeoutFailureDetection()
+    {
+        return this.timeoutFailureDetection;
+    }
+
     @Config("exchange.max-error-duration")
     public ExchangeClientConfig setMaxErrorDuration(Duration maxErrorDuration)
     {
         this.maxErrorDuration = maxErrorDuration;
         return this;
     }
 
+    @Config("exchange.max-retry-count")
+    public ExchangeClientConfig setMaxRetryCount(int maxRetryCount)
```

Comment:
doc pages also should be upated for any new configs

Reply:
fixed.